frontend/udp/parser.go

```diff
@@ -67,6 +66,7 @@ func ParseAnnounce(r Request, allowIPSpoofing bool) (*bittorrent.AnnounceRequest
 	ip := r.IP
 	ipbytes := r.Packet[84:88]
 	if allowIPSpoofing {
+		// TODO(mrd0ll4r) check if this is an empty IP address?
```

Comment:
To respond to your question earlier on IRC. I used logic based on the behavior of the HTTP frontend when removing this before. If you  have enable spoofing and an HTTP announce has `&ip=0.0.0.0`, we don't discount that.